plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorPart.java

```diff
@@ -1079,6 +1095,29 @@ public void run()
             refreshEntityAndAttributes();
         }
     }
+    
+    private class ChangeERDNotationStyleAction extends Action {
+        private final ERDNotationDescriptor notation;
+
+        public ChangeERDNotationStyleAction(ERDNotationDescriptor notation) {
+            super(notation.getName(), AS_CHECK_BOX);
+            this.notation = notation;
+        }
+
+        @Override
+        public boolean isChecked() {
+            if (notation == null || getDiagram().getDiagramNotation() == null) {
```

Comment:
The `notation` field cannot be null. That's why we use `@NotNull`/`@Nullable` annotations.